com.github.unrud.djpdf.yaml

```diff
@@ -891,6 +891,19 @@ modules:
 
   - shared-modules/openjpeg/openjpeg.json
 
+  # libtiff with JPEG support
+  # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1117
+  - name: libtiff
+    buildsystem: cmake-ninja
+    cleanup:
```

Comment:
The `cleanup` section can be removed. It's already in the global section https://github.com/flathub/com.github.unrud.djpdf/blob/cd5f2af3e2d206c7b7ff20242f681dadb32beee4/com.github.unrud.djpdf.yaml#L12-L19

Reply:
1. I copied the fragment from ScanTailor Advanced (I submitted as a PR)
2. I'm of the opinion that each module should cleanup their stuff
3. Even then it doesn't matter on the output.

Reply:
done.